docs/user/index.rst

```diff
@@ -0,0 +1,27 @@
+
```

Comment:
Will this manual entrance be put onto the README file? I think that would be an effective way for user to see refer to the manual.

Reply:
Sure, I forgot to do that. :) Will update README. Thanks!